mlol_client/mlol_client.py

```diff
@@ -803,3 +803,7 @@ def get_user(self) -> Optional[MLOLUser]:
         data = self._api_request(method="GET", url=API_ENDPOINTS["userinfo"])
         if data:
             return MLOLApiConverter.get_user(data)
+
+    @staticmethod
+    def search_library_id(search: str) -> List[(str, int)]:
+        return [(i["name"], i["id"]) for i in requests.get(ENDPOINTS["api"]["portals"]).json() if i["name"].find(search) != -1]
```

Comment:
```suggestion
def find_library_id(query: str) -> List[Tuple[str, int]]:
    response = requests.request(method="GET", url=API_ENDPOINTS["portals"], headers=DEFAULT_API_HEADERS)
    response.raise_for_status()
    if "application/json" in response.headers["Content-Type"]:
        return sorted([(i["name"], i["id"]) for i in response.json() if query in i["name"].lower()])
```

this adds a bit of response validation and returns an alphabetically sorted list